spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkCatalog.java

```diff
@@ -73,6 +74,13 @@
  *   <li><code>warehouse</code> - the warehouse path (Hadoop catalog only)</li>
  *   <li><code>default-namespace</code> - a namespace to use as the default</li>
  *   <li><code>cache-enabled</code> - whether to enable catalog cache</li>
+ *   <li><code>cache.expiration-interval-ms</code> - interval in millis before expiring tables from catalog cache
```

Comment:
I think we don't need these explanation by linking to the javadoc in `CatalogProperties`

Reply:
Sounds good. Will do.

Reply:
Removed the description.